App.js

```diff
@@ -14,6 +16,24 @@ import NewsScreen from './Screens/NewsScreen';
 const Stack = createNativeStackNavigator();
 
 function App() {
+
+  const [loaded, error] = useFonts({
+    Montserrat_500Medium,
+    Montserrat_700Bold,
+  });
+
+  if (!loaded && !error) {
+    return null;
+  }
```

Comment:
Just a nitpick, but in the future when we have a loading screen component, maybe we can return that here instead of null

Reply:
I was going to add splashscreen for the loading screen in the future, I will make changes accordingly when I finish that :)